spec/index.bs

```diff
@@ -1469,29 +1469,42 @@ enum GPUInputStepMode {
 </script>
 
 <script type=idl>
-dictionary GPUVertexAttributeDescriptor {
-    GPUBufferSize offset = 0;
-    required GPUVertexFormat format;
-    required unsigned long shaderLocation;
+dictionary GPUVertexInputDescriptor {
```

Comment:
I'm happy to propose this elsewhere, but I think GPUVertexInputsDescriptor (InputS) would make it easier to keep straight vs Vertex[Buffer,Attribute]Desc.

Reply:
Right after I pushed the previous commit I decided to rename this too. WDYT?

Reply:
"State" is a little vague, but I think it's an improvement.
I think adding "Layout" is more verbose than we need, but it's also fine.
I'm happy with this overall, thanks!